webui/react/src/pages/WorkspaceDetails.tsx

```diff
@@ -1,67 +1,47 @@
-import { Select, Space } from 'antd';
-import React, { useCallback, useEffect, useMemo, useRef, useState } from 'react';
-import { useParams } from 'react-router';
+import { Tabs } from 'antd';
+import React, { useCallback, useRef, useState, useEffect } from 'react';
+import { useParams, useHistory } from 'react-router';
 
-import Grid, { GridMode } from 'components/Grid';
-import GridListRadioGroup, { GridListView } from 'components/GridListRadioGroup';
-import InlineEditor from 'components/InlineEditor';
-import InteractiveTable, { ColumnDef,
-  InteractiveTableSettings,
-  onRightClickableCell } from 'components/InteractiveTable';
-import Link from 'components/Link';
 import Page from 'components/Page';
 import PageNotFound from 'components/PageNotFound';
-import SelectFilter from 'components/SelectFilter';
-import { checkmarkRenderer, GenericRenderer, getFullPaginationConfig,
-  relativeTimeRenderer, stateRenderer, userRenderer } from 'components/Table';
-import Toggle from 'components/Toggle';
 import { useStore } from 'contexts/Store';
 import { useFetchUsers } from 'hooks/useFetch';
 import usePolling from 'hooks/usePolling';
-import useSettings, { UpdateSettings } from 'hooks/useSettings';
-import { paths } from 'routes/utils';
-import { getWorkspace, getWorkspaceProjects, patchProject } from 'services/api';
-import { V1GetWorkspaceProjectsRequestSortBy } from 'services/api-ts-sdk';
+import { getWorkspace } from 'services/api';
 import Message, { MessageType } from 'shared/components/Message';
 import Spinner from 'shared/components/Spinner';
-import { isEqual } from 'shared/utils/data';
-import { ErrorLevel, ErrorType } from 'shared/utils/error';
-import { isNotFound, validateDetApiEnum } from 'shared/utils/service';
-import { ShirtSize } from 'themes';
-import { Project, Workspace } from 'types';
-import handleError from 'utils/error';
-
+import { isNotFound } from 'shared/utils/service';
+import { Workspace } from 'types';
+import { paths } from 'routes/utils';
 import css from './WorkspaceDetails.module.scss';
-import settingsConfig, { DEFAULT_COLUMN_WIDTHS,
-  ProjectColumnName, WhoseProjects, WorkspaceDetailsSettings } from './WorkspaceDetails.settings';
-import ProjectActionDropdown from './WorkspaceDetails/ProjectActionDropdown';
-import ProjectCard from './WorkspaceDetails/ProjectCard';
 import WorkspaceDetailsHeader from './WorkspaceDetails/WorkspaceDetailsHeader';
-
-const { Option } = Select;
+import WorkspaceMembers from './WorkspaceDetails/WorkspaceMembers';
+import WorkspaceProjects from './WorkspaceDetails/WorkspaceProjects';
 
 interface Params {
+  tab: string;
   workspaceId: string;
 }
 
+export enum WorkspaceDetailsTab {
+  Members = 'members',
+  Projects = 'projects'
+}
+
+// Temporary Mock for rbacEnabled functionality
+const rbacEnabled = false;
```

Comment:
We just merged feature switch https://github.com/determined-ai/determined/pull/5036 so you can use `IsOn` instead of the mock